eq-author/src/App/page/Logic/index.js

```diff
@@ -67,36 +67,75 @@ const LogicLink = styled(NavLink)`
   }
 `;
 
+const Badge = styled.span`
+  border-radius: 0.7em;
+  border: 1px solid ${colors.white};
+  background-color: ${colors.red};
+  color: white;
+  padding: 0.1em 0.3em;
+  font-weight: normal;
+  z-index: 2;
+  margin-left: auto;
+  line-height: 1;
+  font-size: 0.9rem;
+  pointer-events: none;
+  width: 20px;
+  height: 20px;
```

Comment:
Could these be changed to em values rather than px, please? If you don't already know, I think 1em = 16px in our app so 20px would be 1.25em.

Reply:
Thanks Matt